streamr-dev/streamr-platform
# Redirect to login for both 401/403 responses.

For some reason in dev, unauthed canvas API calls get served a `403`, but in production, they're served a `401`. This means in prod unauthed canvases end up in a loop spamming the auth endpoint.

This PR makes `loginInterceptor` redirect to login for both `403` & `401` responses.

To test:

* Wait for this to deploy
* Try access a non-public canvas in incognito mode.
* Should redirect to login.
